config-derive/Cargo.toml

```diff
@@ -33,3 +33,4 @@
     json  = []
     toml  = []
     yaml  = []
+    default_trait = []
```

Comment:
You can get rid of this feature, basically I introduced other features because it introduced new deps, here for default trait it doesn't so let's include it by default and delete this feature

Reply:
Thanks for feedback ! 🤗 

I made a feature flag because my PR enforces every struct deriving `twelf::config` to implement `trait std::default::Default`. Due to

```rust
let default_trait_branch = quote! { ::twelf::Layer::DefaultTrait => (::twelf::reexports::serde_json::to_value(<Self as std::default::Default>::default())?,None), };
```

I was thinking that it was a big/breaking change and I just want to be sure that you are ok with that 😉 if you are, I will gladly remove the feature.

Reply:
Oh good catch ! Ok indeed let's keep the feature flag then. I'll merge it and release tomorrow morning